hs/src/PrtlConsts.hs

```diff
@@ -11,4 +11,12 @@ data PrtlConsts =
     minfeeA :: Coin
     -- |The constant factor for the minimum fee calculation
   , minfeeB :: Coin
+    -- |The amount of a key registration deposit
+  , keyDeposit :: Coin
+    -- |The amount of a pool registration deposit
+  , poolDeposit :: Coin
+    -- |The minimum percent refund guarantee
+  , minRefund :: Float
+    -- |The deposit decay rate
+  , decayRate :: Float
```

Comment:
are we really doing `Float` here? why not `Double` or better `Rational`?

Reply:
I like the idea of using `Rational`!

Reply:
ok, now using Rational, works great